Formula/c/clang-uml.rb

```diff
@@ -0,0 +1,104 @@
+class ClangUml < Formula
+  desc "Customizable automatic UML diagram generator for C++ based on Clang"
+  homepage "https://github.com/bkryza/clang-uml"
+  url "https://github.com/bkryza/clang-uml/archive/refs/tags/0.5.3.tar.gz"
+  sha256 "e830363ec510f14cc738c6509107b3f52bc55ececc2e27c068cadb093604e943"
+  license "Apache-2.0"
+  head "https://github.com/bkryza/clang-uml.git", branch: "master"
+
+  depends_on "cmake" => [:build, :test]
+  depends_on "pkg-config" => :build
+  depends_on "llvm"
+  depends_on "yaml-cpp"
+
+  fails_with gcc: "5"
+
+  def llvm
+    deps.map(&:to_formula)
+        .find { |f| f.name.match?(/^llvm(@\d+)?$/) }
+  end
+
+  def install
+    args = %W[
+      -DBUILD_TESTS=OFF
+      -DCMAKE_INSTALL_RPATH=#{rpath(source: libexec/"bin", target: llvm.opt_lib)}
```

Comment:
```suggestion
      -DCMAKE_INSTALL_RPATH=#{rpath(target: llvm.opt_lib)}
```
Missed this one, sorry